ml-agents/mlagents/trainers/torch/distributions.py

```diff
@@ -68,6 +68,9 @@ def pdf(self, value):
     def entropy(self):
         return 0.5 * torch.log(2 * math.pi * math.e * self.std + EPSILON)
 
+    def action_out(self):
```

Comment:
I think the base DistInstance should have a definition for action_out with a docstring